core/src/main/java/org/gbif/pipelines/core/functions/interpretation/MonthInterpreter.java

```diff
@@ -0,0 +1,44 @@
+package org.gbif.pipelines.core.functions.interpretation;
+
+import org.gbif.pipelines.core.functions.interpretation.error.Issue;
+import org.gbif.pipelines.core.functions.interpretation.error.IssueType;
+import org.gbif.pipelines.core.functions.interpretation.error.Lineage;
+import org.gbif.pipelines.core.functions.interpretation.error.LineageType;
+
+import java.time.Month;
+import java.time.format.DateTimeParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * interprets month and add issue and lineages to it.
+ */
+public class MonthInterpreter implements Interpretable<String, Integer> {
+
+  @Override
+  public Integer interpret(String input) throws InterpretationException {
+
+    List<Issue> issues = new ArrayList<>();
+    List<Lineage> lineages = new ArrayList<>();
+
+    Month m;
```

Comment:
next 4 lines can be replaced by return Month.of(Integer.parseInt(input.trim()));